chert/core.py

```diff
@@ -88,7 +88,7 @@
 EXPORT_SRC_EXT = '.md'
 EXPORT_HTML_EXT = '.html'  # some people might prefer .htm
 
-DEV_SERVER_HOST = '0.0.0.0'
+DEV_SERVER_HOST = '127.0.0.1'
```

Comment:
Is it ok for me to switch this to 127.0.0.1 by default?

Reply:
In short, yes, it's fine, because chert has a small userbase and it's overridable via config and/or cli.

The longer version: 0.0.0.0 is not equivalent to 127.0.0.1 because the latter will only respond to requests from the same host. This wouldn't matter much because most chert dev server usage should be local anyways. Except in a containerized world (docker, etc.), 127.0.0.1 will only work for requests inside the container. (Note: I don't use containers, so I'm fine with the change).

To the actual issue, is the problem that the server doesn't listen on the port? I think 0.0.0.0 works for listening on windows. I assume it's a problem with the URL not working in the browser. But if you replace 0.0.0.0 in the browser with localhost or 127.0.0.1, it works. If it's the latter, then I think we just update [this message here](https://github.com/mahmoud/chert/blob/b4a91b5a66ec5f5002d6e67a2f880709e2e11326/chert/core.py#L1114) to be more convenient for Windows users.